### ISSUE TITLE
Why user is allowed to execute unstake multiple times

### ISSUE THREAD

**Issue by shahidnz** [FreeosDAO/freeoscontracts#14]
Staked

Npw calling unstake, multiple times. Every time, no error is shown:

```
06:45:16 06/03/2021 ~/eosio-wallet$ proton push action freeos333333 reguser "[\"${acc}\"]" -p ${acc}@active
executed transaction: f9a57b58e07092bfaa9678749c36eb3baf511e8796488c7b54ae636da4ed8cd0  104 bytes  448 us
#  freeos333333 <= freeos333333::reguser        {"user":"lushalusha"}
>> lushalusha successfully registered
warning: transaction executed locally, but may not be confirmed by the network yet         ]
06:45:30 06/03/2021 ~/eosio-wallet$ proton push action freeos333333 getuser "[\"${acc}\",\"e\"]" -p ${acc}@active
executed transaction: 9d5fab1ccec03fbfa1e9cd4a40a827bd0236776c8e5c12752a892ede827a9bd0  104 bytes  192 us
#  freeos333333 <= freeos333333::getuser        {"user":"lushalusha"}
>> account: lushalusha, registered: 1615009530, type: v, stake: 0.0000 XPR, staked-on: 1615009530, XPR: 10.0000 XPR, liquid: 0.0000 FREEOS, vested: 0.0000 FREEOS, airkey: 0 AIRKEY, iteration: 8, claimed: false
warning: transaction executed locally, but may not be confirmed by the network yet         ]
06:45:48 06/03/2021 ~/eosio-wallet$ proton get currency balance freeos333333 freeosdivide FREEOS
6221.9402 FREEOS
06:46:13 06/03/2021 ~/eosio-wallet$ proton get currency balance eosio.token ${acc} XPR
10.0000 XPR
06:46:23 06/03/2021 ~/eosio-wallet$
06:46:23 06/03/2021 ~/eosio-wallet$ proton get currency balance freeos333333  ${acc} FREEOS
06:46:32 06/03/2021 ~/eosio-wallet$ proton get currency balance eosio.token freeos333333 XPR
109.0000 XPR
06:46:42 06/03/2021 ~/eosio-wallet$ proton push action freeos333333 claim "[\"${acc}\"]" -p $acc@active
executed transaction: 04a62db8cff7af0e21a394b5918cc507cae6efbe0b6afdb821fcaafcec47f8e8  104 bytes  184 us
#  freeos333333 <= freeos333333::claim          {"user":"lushalusha"}
>> user lushalusha has 0.0000 FREEOS which is less than the holding requirement of 613.0000 FREEOS
warning: transaction executed locally, but may not be confirmed by the network yet         ]
06:47:29 06/03/2021 ~/eosio-wallet$
06:50:51 06/03/2021 ~/eosio-wallet$ proton push action freeos333333 unstake "[\"${acc}\"]" -p $acc@active
executed transaction: d725b9145e398d8b0d628766367b79ab78434f7c09f419db874fb7e7dba70062  104 bytes  268 us
#  freeos333333 <= freeos333333::unstake        {"user":"lushalusha"}
warning: transaction executed locally, but may not be confirmed by the network yet         ]
06:52:13 06/03/2021 ~/eosio-wallet$ proton get currency balance eosio.token ${acc} XPR
10.0000 XPR
06:52:31 06/03/2021 ~/eosio-wallet$ proton get currency balance freeos333333  ${acc} FREEOS
06:52:43 06/03/2021 ~/eosio-wallet$ proton get currency balance eosio.token freeos333333 XPR
109.0000 XPR
06:52:54 06/03/2021 ~/eosio-wallet$ proton push action freeos333333 unstake "[\"${acc}\"]" -p $acc@active
executed transaction: 4ee62860f2aad280ef304b689317dcd109f6fdc399270a0d5d7fe521a7a11638  104 bytes  272 us
#  freeos333333 <= freeos333333::unstake        {"user":"lushalusha"}
warning: transaction executed locally, but may not be confirmed by the network yet         ]
06:53:04 06/03/2021 ~/eosio-wallet$ proton push action freeos333333 unstake "[\"${acc}\"]" -p $acc@active
executed transaction: b2b7fbfc9e552e997a515c36b220cc2737cf8d5f46c372e2f0719b43dfeee9bc  104 bytes  252 us
#  freeos333333 <= freeos333333::unstake        {"user":"lushalusha"}
warning: transaction executed locally, but may not be confirmed by the network yet         ]
06:53:15 06/03/2021 ~/eosio-wallet$ proton push action freeos333333 unstake "[\"${acc}\"]" -p $acc@active
executed transaction: 6d45808d24b8870b812a0865cab1a7ffdb5b05e7cfa0c99bae603336685eee5e  104 bytes  267 us
#  freeos333333 <= freeos333333::unstake        {"user":"lushalusha"}
warning: transaction executed locally, but may not be confirmed by the network yet         ]
```

**Comment by tommccann777** [comment]
Good catch! I'll look into this.

**Comment by tommccann777** [comment]
This has been fixed in v0.330 which was deployed 10.40 tonight (7 Mar 2021).
Please retest. Thanks.
